cluster/addons/limit-range/limit-range.json

```diff
@@ -0,0 +1,20 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "LimitRange",
+  "metadata": {
+    "name": "limits"
+  },
+  "spec": {
+    "limits": [
+    {
+      "type": "Container",
+      "min": {
+        "cpu": "400m"
```

Comment:
Ahh, forgot pushing the latest version with 100m. 400m just for my own test, and you can see my pr description is using 100m which is the latest one. :-p